base_local_planner/include/base_local_planner/obstacle_cost_function.h

```diff
@@ -83,6 +83,9 @@ class ObstacleCostFunction : public TrajectoryCostFunction {
   bool sum_scores_;
   //footprint scaling with velocity;
   double max_forward_inflation_, max_sideward_inflation_, scaling_speed_;
+
+  ros::Subscriber sub_;
```

Comment:
```suggestion
  ros::Subscriber max_sideward_inflation_scale_sub_;
```
or better w/o max_  see below

Reply:
Fixed